public/bustracker.html

```diff
@@ -1,282 +0,0 @@
-
```

Comment:
Some pngs that didn't seem to be being used (except maybe in the ruby gui) are gone.

This file is too big and I have broken it up